protostar/starknet_gateway/gateway_facade.py

```diff
@@ -184,6 +186,50 @@ async def declare(
             transaction_hash=result.transaction_hash,
         )
 
+    async def call(
+        self,
+        address: AddressRepresentation,
+        function_name: str,
+        inputs: Optional[GatewayFacadeSupportedInputType] = None,
```

Comment:
```suggestion
        inputs: Optional[FunctionInputs] = None,
```

wdyt?

Reply:
and please, move the definition of this type to this PR from #566

Reply:
Renamed it `ContractFunctionInputType` in the ProtostarFixture PR.